lib/functions/function_plot_measures.R

```diff
@@ -0,0 +1,31 @@
+plot_measures <- function(
+    data,
+    title = NULL,
+    x_label = NULL,
+    y_label = NULL) {
+
+  # Create plot
+  plot <- ggplot(
+    data,
+    aes(
+      x = interval_end,
+      y = numerator,
+      color = measure,
+      group = measure
```

Comment:
To make this function more robust we should use _tidyeval_ here, see https://ggplot2.tidyverse.org/reference/tidyeval.html. You'd need to add more arguments to the function where we can spoecify the variable names from the measures file.

This might not be needed as all measures files will come with exactly these variable names. We could also decide to check the data for the presence of these variable names with something like this. This is just checking for one variable and would have to get extended for all the variables the function needs:

```
  # Check variables exist in data
  if ("numerator" %in% names(data) == FALSE {
    stop("Data must have a column with the name 'numerator'")
  }
```